DigitalLearningSolutions.Web/appsettings.Development.json

```diff
@@ -17,5 +17,5 @@
     "PricingPage": true,
     "RefactoredFindYourCentrePage": true
   },
-  "LearningHubOpenAPIBaseUrl": "https://uks-learninghubnhsuk-openapi-test.azurewebsites.net"
+  "LearningHubOpenAPIBaseUrl": "https://uks-learninghubnhsuk-openapi-dev.azurewebsites.net"

```

Comment:
This was intentionally pointing to Test - Test is in sync with the Findwise collection (more or less), whereas Dev isn't, so the search endpoint gives results which are much more consistent with Prod if we point to Test.

Reply:
Ah whoops, I didn't mean to commit this change